lib/manager/maven/index.ts

```diff
@@ -3,7 +3,7 @@ import * as datasourceMaven from '../../datasource/maven';
 import * as mavenVersioning from '../../versioning/maven';
 
 export { extractAllPackageFiles } from './extract';
-export { updateDependency } from './update';
+export { updateDependency, bumpPackageVersion } from './update';
```

Comment:
This snippet violates ESLint `sort-imports`
```suggestion
export { bumpPackageVersion, updateDependency } from './update';
```